poll.go

```diff
@@ -12,7 +12,7 @@ import (
 	"time"
 )
 
-var statuses = [5]string{"UNKNOWN", "DEPLOYED", "DELETED", "SUPERSEDED", "FAILED"}
+var statuses = [5]string{"unknown", "deployed", "deleted", "superseded", "failed"}
```

Comment:
Are we comparing this values directly with what is returned by helm? Should we make them lower case as you did, but also do a "toLowerCase" on the helm response? This way would work for both helm version